cider/exec/nextgen/jitlib/llvmjit/LLVMJITModule.h

```diff
@@ -53,7 +53,8 @@ class LLVMJITModule final : public JITModule {
 
  public:
   explicit LLVMJITModule(const std::string& name,
-                         bool should_copy_runtime_module = false);
+                         bool should_copy_runtime_module = false,
+                         CompilationOptions co = CompilationOptions{});
 
   LLVMJITModule(const std::string& name, CompilationOptions co);
```

Comment:
we should merge these constructors: `LLVMJITModule(const std::string& name, CompilationOptions co  = CompilationOptions{}, bool should_copy_runtime_module = false);`

Reply:
Need to add `compilationOptions ` field getter method?  better for some expr codegen func/jfunction usage.